src/e2e-test/features/salesforcebatchsource/RunTime.feature

```diff
@@ -0,0 +1,128 @@
+# Copyright © 2022 Cask Data, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+
+@SalesforceSalesCloud
+@SFBatchSource
+@Smoke
+@Regression
+
+Feature: Salesforce Batch Source - Run time Scenarios
+
+  @BATCH-TS-SF-RNTM-01 @BQ_SINK_TEST
+  Scenario: Verify user should be able to preview and deploy the pipeline when plugin is configured for SObject Name
+    When Open Datafusion Project to configure pipeline
+    And Select data pipeline type as: "Batch"
+    And Select plugin: "Salesforce" from the plugins list as: "Source"
+    And Navigate to the properties page of plugin: "Salesforce"
+    And fill Authentication properties for Salesforce Admin user
+    And Enter input plugin property: "referenceName" with value: "Reference"
+    And Enter input plugin property: "sObjectName" with value: "sobject.lead"
+    Then Validate "Salesforce" plugin properties
+    And Capture the generated Output Schema
+    And Close the Plugin Properties page
+    And Select Sink plugin: "BigQueryTable" from the plugins list
+    And Navigate to the properties page of plugin: "BigQuery"
+    Then Replace input plugin property: "project" with value: "projectId"
+    Then Enter input plugin property: "datasetProject" with value: "projectId"
+    Then Enter input plugin property: "referenceName" with value: "BQReferenceName"
+    Then Enter input plugin property: "dataset" with value: "dataset"
+    Then Enter input plugin property: "table" with value: "bqTargetTable"
+    Then Validate "BigQuery" plugin properties
+    Then Close the Plugin Properties page
+    Then Connect plugins: "Salesforce" and "BigQuery" to establish connection
+    And Preview and run the pipeline
+    Then Wait till pipeline preview is in running state
+    Then Open and capture pipeline preview logs
+    Then Verify the preview run status of pipeline in the logs is "succeeded"
+    And Close the pipeline logs
+    And Close the preview
+    And Save and Deploy Pipeline
+    And Run the Pipeline in Runtime
+    And Wait till pipeline is in running state
+    And Open and capture logs
+    And Verify the pipeline status is "Succeeded"
+    And Close the pipeline logs
```

Comment:
Can we verify here if the data transferred from the salesforce side is same as the data received in the BQ sink? Please do so for the below scenarios too

Reply:
The team is working on the data validation for all the 4 plugins, I believe they will be raising a separate PR for that. Should we add that change in this PR as well ?